src/views/components/create-item-form.jsx

```diff
@@ -21,13 +22,18 @@ export default class CreateItemForm extends ListenerComponent {
     changeModel(e) {
         this.setState({
             modelAddress: e.target.value,
-            models: this.state.models
         });
     }
 
     submit(e) {
         e.preventDefault();
         createItem(this.state.modelAddress);
+
+        searchModel(this.state.modelAddress);
```

Comment:
This is asynchronous. The model won't update until a response is received from the server.